test/helpers.js

```diff
@@ -3,7 +3,7 @@ var tmp = require('os-tmpdir')
 var path = require('path')
 var rimraf = require('rimraf')
 var randombytes = require('randombytes')
-var Osm = require('osm-p2p-mem')
+var Osm = require('osm-p2p')
```

Comment:
testing with actual osm-p2p, which includes leveldb instead of with memory will allow us to have a more real life scenario

Reply:
since the osm-p2p cleanup is giving you trouble, I suggest getting it passing with osm-p2p-mem first and then addressing the fs issues. I just published a major semver of osm-p2p-mem that uses kappa